### PR TITLE
Video stream metrics

### PR DESCRIPTION
**Pull request by karimkozman**
It always gives me `'Cannot read property 'getActiveStream' of undefined'` when I do `this.player.getActiveStream().getStreamInfo();` even when the player is initialized...

### CODE REVIEW

File: src/components/dashjs-statistics/dashjs-statistics.tsx [hunk 1]
```diff
@@ -1,296 +1,416 @@
...
 })
-
 export class DashjsStatistics {
+  @Prop()
+  video_data: any;
+  @Prop()
+  audio_data: any;
+
+  @State()
+  videoDisable: string = 'Disable';
```
Comment (DanielHabenicht) [inline src/components/dashjs-statistics/dashjs-statistics.tsx:17]: please use booleans

File: src/components/dashjs-statistics/dashjs-statistics.tsx [hunk 2]
```diff
@@ -1,296 +1,416 @@
...
+
+  @State()
+  audioDisable: string = 'Disable';
+
+  // DashMetrics properties
+  @State()
+  videoBufferLevel: number;
```
Comment (DanielHabenicht) [inline src/components/dashjs-statistics/dashjs-statistics.tsx:24]: those metrics might change, should i generate a JSON with all metrics from the documentation so you can do this on a higher level, without writing down every single metric?

File: src/components/dashjs-statistics/dashjs-statistics.tsx (hunk 1 above)
Comment (karimkozman) [reply src/components/dashjs-statistics/dashjs-statistics.tsx:17]: @hieu2010 should I change these to booleans?

File: src/components/dashjs-statistics/dashjs-statistics.tsx (hunk 2 above)
Comment (karimkozman) [reply src/components/dashjs-statistics/dashjs-statistics.tsx:24]: But I would still need to declare a state for each right? As some function are from `dashMetrics` and `dashAdapter` and each one takes different parameters as you can see in `streamMetrics()`

Comment (DanielHabenicht) [reply src/components/dashjs-statistics/dashjs-statistics.tsx:24]: no, more like a map of these values

Comment (karimkozman) [reply src/components/dashjs-statistics/dashjs-statistics.tsx:24]: Okay👍🏼

Comment (DanielHabenicht) [reply src/components/dashjs-statistics/dashjs-statistics.tsx:24]: if you want to incorporate it now just use an array with those properties as strings... then I can plug in the generated JSON later.

File: src/components/dashjs-statistics/dashjs-statistics.tsx (hunk 1 above)
Comment (hieu2010) [reply src/components/dashjs-statistics/dashjs-statistics.tsx:17]: yes

**Pull request merged**